api/src/test/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandlerTest.java

```diff
@@ -0,0 +1,268 @@
+package org.openmrs.module.fhir.api.diagnosticreport.handler;
+
+import org.hl7.fhir.dstu3.model.Attachment;
+import org.hl7.fhir.dstu3.model.DiagnosticReport;
+import org.hl7.fhir.dstu3.model.Reference;
+
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Ignore;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.openmrs.Concept;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterRole;
+import org.openmrs.EncounterType;
+import org.openmrs.Patient;
+import org.openmrs.PatientIdentifier;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.Person;
+import org.openmrs.PersonAddress;
+import org.openmrs.PersonName;
+import org.openmrs.Provider;
+import org.openmrs.Location;
+import org.openmrs.api.EncounterService;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.context.Context;
+
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.openmrs.module.fhir.api.diagnosticreport.handler.LaboratoryHandler;
+
+import java.util.Date;
+import java.util.Set;
+import java.util.List;
+import java.util.LinkedHashSet;;
+import java.util.TreeSet;;
+
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.sameInstance;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.MockitoAnnotations.initMocks;
+import static org.powermock.api.mockito.PowerMockito.when;
+
```

Comment:
I think repeated and unused imports can be removed.

Reply:
I was pretty sure that the PR needed some changes so I didn't bother with them earlier, but I guess now's a good time to clean up. Thanks for pointing it out @haripriya999 :)